src/component/registry/bpmn-elements-registry.ts

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { ensureIsArray } from '../parser/json/converter/utils';
+import { ensureIsArray } from '../helpers/array-utils';
```

Comment:
⚠️ there is a remaining TODO about the move of this function in getElementsByIds
It should be removed as this is managed by this PR 😸

Reply:
I need to create a array utils for #1043